test/helpers/field_enums.py

```diff
@@ -18,8 +24,8 @@ class BlockFields(NamedFields):
     timestamp = 3
 
     @classmethod
-    def select_query(cls, table="blocks"):
-        return super().select_query(table)
+    def select_query(cls, table="blocks", prefix=False):
```

Comment:
I feel responsible for you having to do this, my apologies. Let's leave it like this for this one but I think in the future it would make more sense if we break `NamedFields` up into 3 module-level functions and call them directly in the enum classes (i.e. stop inheriting).